docker/client.Dockerfile.dev

```diff
@@ -7,7 +7,7 @@ RUN npm i -g npm@7.20.0
 
 # set working directory
 WORKDIR /app
-
+COPY ./.version ./.version
```

Comment:
You will have to update the other dockerfile setups as well. 
Will it be simpler to infer from `/metadata` call to the server which is being done [here](https://github.com/ToolJet/ToolJet/blob/develop/frontend/src/App/App.jsx#L45)?